bundles/org.openhab.binding.comfoair/src/main/java/org/openhab/binding/comfoair/internal/ComfoAirCommandType.java

```diff
@@ -0,0 +1,862 @@
+            new String[] { "analog#analog3Negative", "analog#analog3Min", "analog#analog3Max", "analog#analog3Value",
+                    "analog#analog3Volt" },
+            Constants.REQUEST_GET_ANALOGS, Constants.REPLY_GET_ANALOGS, new int[] { 1 }, 0x04),
+    ANALOG4_MODE("analog#analog4Mode", DataTypeBoolean.class, new int[] { 0x08 }, Constants.REQUEST_SET_ANALOGS, 19, 1,
+            new String[] { "analog#analog4Negative", "analog#analog4Min", "analog#analog4Max", "analog#analog4Value",
+                    "analog#analog4Volt" },
+            Constants.REQUEST_GET_ANALOGS, Constants.REPLY_GET_ANALOGS, new int[] { 1 }, 0x08),
+    RF_MODE("analog#RFMode", DataTypeBoolean.class, new int[] { 0x10 }, Constants.REQUEST_SET_ANALOGS, 19, 1,
+            new String[] { "analog#RFNegative", "analog#RFMin", "analog#RFMax", "analog#RFValue" },
+            Constants.REQUEST_GET_ANALOGS, Constants.REPLY_GET_ANALOGS, new int[] { 1 }, 0x10),
+    ANALOG1_NEGATIVE("analog#analog1Negative", DataTypeBoolean.class, new int[] { 0x01 }, Constants.REQUEST_SET_ANALOGS,
+            19, 2, new String[] {}, Constants.REQUEST_GET_ANALOGS, Constants.REPLY_GET_ANALOGS, new int[] { 2 }, 0x01),
+    ANALOG2_NEGATIVE("analog#analog2Negative", DataTypeBoolean.class, new int[] { 0x02 }, Constants.REQUEST_SET_ANALOGS,
+            19, 2, new String[] {}, Constants.REQUEST_GET_ANALOGS, Constants.REPLY_GET_ANALOGS, new int[] { 2 }, 0x02),
+    ANALOG3_NEGATIVE("analog#analog3Negative", DataTypeBoolean.class, new int[] { 0x04 }, Constants.REQUEST_SET_ANALOGS,
+            19, 2, new String[] {}, Constants.REQUEST_GET_ANALOGS, Constants.REPLY_GET_ANALOGS, new int[] { 2 }, 0x04),
+    ANALOG4_NEGATIVE("analog#analog4Negative", DataTypeBoolean.class, new int[] { 0x08 }, Constants.REQUEST_SET_ANALOGS,
+            19, 2, new String[] {}, Constants.REQUEST_GET_ANALOGS, Constants.REPLY_GET_ANALOGS, new int[] { 2 }, 0x08),
+    RF_NEGATIVE("analog#RFNegative", DataTypeBoolean.class, new int[] { 0x10 }, Constants.REQUEST_SET_ANALOGS, 19, 2,
+            new String[] {}, Constants.REQUEST_GET_ANALOGS, Constants.REPLY_GET_ANALOGS, new int[] { 2 }, 0x10),
+    ANALOG1_MIN("analog#analog1Min", DataTypeNumber.class, Constants.REQUEST_SET_ANALOGS, 19, 3, new String[] {},
+            Constants.REQUEST_GET_ANALOGS, Constants.REPLY_GET_ANALOGS, new int[] { 3 }),
+    ANALOG1_MAX("analog#analog1Max", DataTypeNumber.class, Constants.REQUEST_SET_ANALOGS, 19, 4, new String[] {},
+            Constants.REQUEST_GET_ANALOGS, Constants.REPLY_GET_ANALOGS, new int[] { 4 }),
+    ANALOG1_VALUE("analog#analog1Value", DataTypeNumber.class, Constants.REQUEST_SET_ANALOGS, 19, 5, new String[] {},
+            Constants.REQUEST_GET_ANALOGS, Constants.REPLY_GET_ANALOGS, new int[] { 5 }),
+    ANALOG2_MIN("analog#analog2Min", DataTypeNumber.class, Constants.REQUEST_SET_ANALOGS, 19, 6, new String[] {},
```

Comment:
Just like my `new int[0]` constant suggestion, you should create a `new String[0]` constant as well.
```suggestion
    ANALOG2_MIN("analog#analog2Min", DataTypeNumber.class, Constants.REQUEST_SET_ANALOGS, 19, 6, new String[0],
```